notebooks/train_model.ipynb

```diff
@@ -119,18 +133,18 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "\u001b[1m47/47\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m1s\u001b[0m 18ms/step - accuracy: 0.0000e+00 - loss: 0.0000e+00 - val_accuracy: 0.0000e+00 - val_loss: 0.0000e+00\n",
+      "\u001b[1m47/47\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m0s\u001b[0m 11ms/step - accuracy: 0.0000e+00 - loss: 0.0000e+00 - val_accuracy: 0.0000e+00 - val_loss: 0.0000e+00\n",
       "Epoch 3/3\n",
-      "\u001b[1m47/47\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m506s\u001b[0m 11s/step - accuracy: 0.9749 - loss: 0.1335 - val_accuracy: 0.5000 - val_loss: 7.8295\n"
+      "\u001b[1m47/47\u001b[0m \u001b[32m━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[37m\u001b[0m \u001b[1m552s\u001b[0m 12s/step - accuracy: 0.9731 - loss: 0.0932 - val_accuracy: 0.5000 - val_loss: 142.4934\n"
```

Comment:
the model seems to fit well on training data, with over 97%, and then the validation is a bit scary just 50%, so that basically shows a bit of overfitting. 
so basically there is poor genralisation to unseen data.